.github/workflows/rust.yml

```diff
@@ -54,8 +55,30 @@ jobs:
     - uses: Swatinem/rust-cache@v1
     - uses: actions/checkout@v2
 
+    - name: Checkout RedisJSON
+      uses: actions/checkout@v2
+      with:
+        repository: "RedisJSON/RedisJSON"
+        path: "./__ci/redis-json"
+        set-safe-directory: false
+
+    - name: Compile RedisJSON
+      run: |
+        # copy current Cargo.toml
```

Comment:
Can we add a small comment above this section briefly explaining why the steps here look so crazy? I'm not too concerned about it and don't think it's necessary to wrestle with it further, but I'm hoping that we'll be able to solve the problem here a bit more elegantly in the future.

Reply:
Yeah this is something that `cargo` would need to change, telling it *not* to go up a level or 2 to check for workspaces would be nice to be able to disable